### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by danielduhh** [spatialdev/osmquality#12]
Something is up with the module imports.

This error is thrown when running:

```bash
python3 src/mqm/mqm_tool.py /path/to/folder
```

Error
```bash
Traceback (most recent call last):
  File "/Users/danielbaah/github/osmquality/src/mqm/mqm_tool.py", line 6, in <module>
    from .kd_tree import kdTree
ModuleNotFoundError: No module named '__main__.kd_tree'; '__main__' is not a package
```

**Comment by nachtm** [comment]
Closed by #13?

**Comment by danielduhh** [comment]
Yes! ty
